<ui version="4.0" >
 <class>PrefPerformance</class>
 <widget class="QWidget" name="PrefPerformance" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Performance</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="priority_group" >
         <property name="title" >
          <string>Priority</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="priority_intro_label" >
            <property name="text" >
             <string>Select the priority for the MPlayer process.</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="priority_label" >
              <property name="text" >
               <string>Priorit&amp;y:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>priority_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="priority_combo" >
              <item>
               <property name="text" >
                <string>realtime</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>high</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>abovenormal</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>normal</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>belownormal</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>idle</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>201</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="framedrop_check" >
         <property name="text" >
          <string>&amp;Allow frame drop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hardframedrop_check" >
         <property name="text" >
          <string>Allow &amp;hard frame drop (can lead to image distortion)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>H.264</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Loop &amp;filter</string>
            </property>
            <property name="buddy" >
             <cstring>loopfilter_combo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="loopfilter_combo" />
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Synchronization</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="autosync_check" >
            <property name="text" >
             <string>Audio/&amp;video auto synchronization</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="factor_label" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Fact&amp;or:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>autosync_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="autosync_spin" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="maximum" >
               <number>1000</number>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&amp;Fast audio track switching</string>
           </property>
           <property name="buddy" >
            <cstring>fast_audio_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TristateCombo" name="fast_audio_combo" />
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="fast_chapter_check" >
         <property name="text" >
          <string>Fast &amp;seek to chapters in dvds</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="idx_check" >
         <property name="text" >
          <string>C&amp;reate index if needed</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>&amp;Cache</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="3" column="3" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>211</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>KB</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>KB</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QSpinBox" name="cache_dvds_spin" >
         <property name="maximum" >
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Cache for &amp;DVDs:</string>
         </property>
         <property name="buddy" >
          <cstring>cache_dvds_spin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4" >
        <widget class="QLabel" name="textLabel1_12" >
         <property name="text" >
          <string>Setting a cache may improve performance on slow media</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Cache for &amp;local files:</string>
         </property>
         <property name="buddy" >
          <cstring>cache_files_spin</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Cache for &amp;streams:</string>
         </property>
         <property name="buddy" >
          <cstring>cache_streams_spin</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>211</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>211</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>KB</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>351</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="cache_files_spin" >
         <property name="maximum" >
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QSpinBox" name="cache_streams_spin" >
         <property name="maximum" >
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TristateCombo</class>
   <extends>QComboBox</extends>
   <header>tristatecombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>priority_combo</tabstop>
  <tabstop>framedrop_check</tabstop>
  <tabstop>hardframedrop_check</tabstop>
  <tabstop>autosync_check</tabstop>
  <tabstop>autosync_spin</tabstop>
  <tabstop>fast_chapter_check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>autosync_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>factor_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>270</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel" >
     <x>90</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autosync_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>autosync_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>270</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel" >
     <x>147</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
